Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Dockerfile to build a container with the host-check script
```

Comment:
It should be clearer that this is for host-check ... unfortunately there's no standard way of naming Dockerfiles (except just `Dockerfile`), although `Dockerfile.xxx` is pretty common.

I'd suggest renaming to `Dockerfile.host-check`, given it wouldn't make sense to put host-check in its own subdirectory.